test/rebar_test_utils.erl

```diff
@@ -163,24 +156,18 @@ create_config(_AppDir, ConfFilename, Contents) ->
 %% @doc Util to create a random variation of a given name.
 create_random_name(Name) ->
     random_seed(),
-    Name ++ erlang:integer_to_list(?random:uniform(1000000)).
+    Name ++ erlang:integer_to_list(rand:uniform(1000000)).
 
 %% @doc Util to create a random variation of a given version.
 create_random_vsn() ->
     random_seed(),
-    lists:flatten([erlang:integer_to_list(?random:uniform(100)),
-                   ".", erlang:integer_to_list(?random:uniform(100)),
-                   ".", erlang:integer_to_list(?random:uniform(100))]).
+    lists:flatten([erlang:integer_to_list(rand:uniform(100)),
+                   ".", erlang:integer_to_list(rand:uniform(100)),
+                   ".", erlang:integer_to_list(rand:uniform(100))]).
 
--ifdef(rand_only).
 random_seed() ->
```

Comment:
we can delete this and its call on line 158

Reply:
026f90f